docs/color-system.md

```diff
@@ -0,0 +1,83 @@
+# Color system
```

Comment:
Out of interest, how do you get to these docs from our repo root?

Reply:
Good spot! Right now, nothing links to it.

I'll fix that.